bot/commands/Configuration/alias.js

```diff
@@ -0,0 +1,33 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        super(...args, {
+            runIn: ['text'],
+            cooldown: 2,
+            subcommands: true,
+            description: 'Enable and Disable certain command aliases in your guild. Admins only.',
+            usage: '<enable|disable> <alias:str>',
+            usageDelim: ' ',
+            permissionLevel: 7
+        });
+    }
+
+    async enable(msg, alias) {
```

Comment:
Will enable and disable add/remove the string to `disabledAliases` in `config/Schemas/GuildSchema.js`?